src/main/resources/META-INF/archetype-post-generate.groovy

```diff
@@ -239,7 +239,8 @@ def cleanUpFrontendModule(frontendModules, optionFrontendModule, rootPom, rootDi
     // Not generating SPA: Delete SPA-specific files
     if (optionFrontendModule != "angular" && optionFrontendModule != "react") {
         // Delete app component
-        assert new File("$appsFolder/components/structure/spa").deleteDir()
+        assert new File("$appsFolder/components/spa").deleteDir()
```

Comment:
This looks unrelated to the rest of the changes.

Reply:
Nice find. This is indeed just a bug  I should separate in another PR. il revert it for now.. we can do it later somehow